### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by tobiastykvart**
Keeping the dependencies up to date and removed some unused/unnecessary packages.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -12,7 +13,6 @@ module.exports = {
 
     // Community addons
     'storybook-mobile',
-    'storybook-addon-mdx-embed',
```
Comment (tobiastykvart) [inline .storybook/main.js:15]: Unused dependency.

File: .storybook/main.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
   stories: ['../src/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
+  staticDirs: ['../public'],
```
Comment (tobiastykvart) [inline .storybook/main.js:3]: Setting the static directory in `package.json` was deprecated, it has to be set here instead.

File: .eslintrc [hunk 3]
```diff
@@ -34,7 +34,6 @@
     "jest/no-identical-title": "error",
     "jest/prefer-to-have-length": "warn",
     "jest/valid-expect": "error",
-    "valid-jsdoc": "off",
```
Comment (tobiastykvart) [inline .eslintrc:37]: I'm removing jsdoc, has been mostly unused and we can document components in better ways.

File: package.json [hunk 4]
```diff
@@ -15,11 +15,11 @@
   "homepage": "https://github.com/factiverse/storybook-design-system#readme",
   "scripts": {
     "generate": "node ./util/create-component",
-    "storybook": "start-storybook -p 6006 -s public",
```
Comment (tobiastykvart) [inline package.json:18]: Deprecated, see above.

File: package.json [hunk 5]
```diff
@@ -15,11 +15,11 @@
...
+    "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "build-storybook-docs": "build-storybook  --docs",
+    "build-storybook-docs": "build-storybook --docs",
     "chromatic": "npx chromatic --project-token=e8130cbb59b8 --exit-once-uploaded",
-    "deploy-storybook": "storybook-to-ghpages",
+    "deploy": "storybook-to-ghpages",
```
Comment (tobiastykvart) [inline package.json:22]: deploy command deploys to GitHub pages in other repos, should be called the same here.

File: package.json [hunk 6]
```diff
@@ -76,17 +76,10 @@
   "main": "build/index.js",
   "module": "build/esm/index.js",
   "types": "build/index.d.ts",
-  "overrides": {
```
Comment (tobiastykvart) [inline package.json:79]: Removed the unnecessary dependency mdx-embed.

**Comment by withchristopher** [comment]
Is this PR still valid with the changes?

**Pull request closed without merging**